go/exocom/src/exocom/exocom.go

```diff
@@ -9,16 +9,20 @@ import (
 
 	"github.com/Originate/exocom/go/exocom/src/connection"
 	"github.com/Originate/exocom/go/exocom/src/routing"
+	"github.com/Originate/exocom/go/exocom/src/security"
 	"github.com/Originate/exocom/go/exocom/src/types"
 	"github.com/Originate/exocom/go/structs"
 	"github.com/gorilla/websocket"
 )
 
 // ExoCom is the top level message broadcaster
 type ExoCom struct {
+	securityManager   *security.Manager
+	authTable         map[string]structs.Message
 	connectionManager *connection.Manager
 	deregisterChannel chan string
 	errorChannel      chan error
+	hasSecurity       bool
```

Comment:
Thoughts on passing this through to the security manager? Then we don't need to store it here and the security manager will return the passed in message when this is false.